src/cli/index.ts

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env node
+import program from 'commander';
+
+import main from '../main';
+import collect from './collect';
+
+program.version(require('../../package.json').version);
+
+program.option('--ignore <path>', 'Specify path of ignore file', collect, []);
```

Comment:
Is the ‘ignore’ a file containing ignore patterns or a file to ignore?

Reply:
changed,  it is `--ignore-pattern`